qa/1689.out

```diff
@@ -141,11 +159,23 @@ pmproxy.discover.throttled_changed_callbacks PMID: 4.5.18 [filesystem changed ca
 Help:
 number of throttled filesystem change callbacks
 
-pmproxy.pid PMID: 4.1.0 [pmproxy PID]
+pmproxy.mem.datasz PMID: 4.1.5 [server virtual data size]
+    Data Type: 64-bit unsigned int  InDom: PM_INDOM_NULL 0xffffffff
+    Semantics: instant  Units: Kbyte
+Help:
+server process data memory size, returned from sbrk(2)
+
+pmproxy.mem.maxrss PMID: 4.1.4 [server maximum RSS]
+    Data Type: 64-bit unsigned int  InDom: PM_INDOM_NULL 0xffffffff
+    Semantics: instant  Units: Kbyte
+Help:
+server process maximum resident set memory size
+
+pmproxy.pid PMID: 4.1.0 [server PID]
     Data Type: 32-bit unsigned int  InDom: PM_INDOM_NULL 0xffffffff
     Semantics: discrete  Units: none
 Help:
-PID for the current pmproxy invocation
+PID for the current server invocation
```

Comment:
'PID' -> 'Identifier'
'server invocation' -> 'process'
... really this is all not super helpful long text - maybe just have shorttext for this one as its pretty obvious?